driver-adapters-wasm/pg-cf-basic/package.json

```diff
@@ -16,7 +16,6 @@
     "@prisma/adapter-pg": "5.8.0-dev.10",
     "@prisma/client": "5.8.0-dev.10",
     "add": "^2.0.6",
-    "pg": "8.11.3",
-    "pg-cloudflare": "1.1.1"
```

Comment:
I think this is not needed in the end because pnpm and npm install optional peer dependencies

Reply:
I think that was just a copy/paste from other projects that already exist.